AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -264,17 +289,13 @@ - (void)fetchData
   {
     ASDN::MutexLocker l(_videoLock);
 
-    AVPlayerItem *playerItem = [self constructPlayerItem];
-    self.currentItem = playerItem;
-
-    if (_player != nil) {
-      [_player replaceCurrentItemWithPlayerItem:playerItem];
-    } else {
-      self.player = [AVPlayer playerWithPlayerItem:playerItem];
-    }
-
-    if (_placeholderImageNode.image == nil) {
-      [self generatePlaceholderImage];
+    if (_asset != nil) {
+      NSArray *requestedKeys = @[ @"playable" ];
+      [_asset loadValuesAsynchronouslyForKeys:requestedKeys completionHandler:^{
+          dispatch_async(dispatch_get_main_queue(), ^{
+            [self prepareToPlayAsset:_asset withKeys:requestedKeys];
```

Comment:
A check may be needed to ensure _asset hasn't changed in the time between this request was made, and the asynchronous completionHandler is invoked.